### ISSUE TITLE
DevTools attempts to clone uncloneable objects

### ISSUE THREAD

**Issue by oliverwoodings** [jhollingworth/marty#165]
I am dispatching the state of a react-router Router however it has some instances of a Routes attached on it which for some reason Marty is unable to clone and post into the DevTools frame. Not quite sure how we could handle this!

![screen shot 2015-03-06 at 16 59 58](https://cloud.githubusercontent.com/assets/621323/6529710/7ff0d204-c422-11e4-878d-ea3bb994b841.png)

**Comment by oliverwoodings** [comment]
Just realised this is a copy of https://github.com/jhollingworth/marty/issues/109. Seems like this hasn't gone away fully? I'm using Marty 0.8.12 with the latest dev tools. It only happens when using Immutable state in my stores.

**Comment by mapreal19** [comment]
Experienced this as well in version 0.8.13

**Comment by jhollingworth** [comment]
Currently working on a re-write of DevTools for v0.9 which will include a fix for this issue

**Comment by jhollingworth** [comment]
In Marty DevTools v0.9 (Will be released once React Router v0.13 is done) we've got the concept of pluggable serialisers that allow you to define your own serialisation process (This was actually @oliverwoodings idea)

```js
MartyDevTools.registerSerializer({
  canSerialize: function (obj) { return true; },
  serialize: function (obj) { ... }
});
```
